3483 - Fix fileupload alignment with forms compact (#3522)
infor-design/enterprise: docs/CHANGELOG.md

- `[Form Compact Layout]` Added support for `form-compact-layout` the remaining components. ([#3008](https://github.com/infor-design/enterprise/issues/3329))
- `[Dropdown]` Fixed a bug that was causing the `selectValue()` method not to update the visual display of the in-page Dropdown element. ([#3432](https://github.com/infor-design/enterprise/issues/3432))
- `[Forms]` Fixed an issue where radio group was overlapping fields. ([#3466](https://github.com/infor-design/enterprise/issues/3466))

> - `[Forms Compact]` Fixed an issue where fileupload was not aligning and the checkboxes were not aligning for RTL in Uplift/Vivrant theme. ([#3483](https://github.com/infor-design/enterprise/issues/3483))

Reviewer: ```suggestion
- `[Forms Compact]` Fixed an issue where fileupload was misaligned in RTL mode in uplift theme. ([#3483](https://github.com/infor-design/enterprise/issues/3483))
```
